org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionHandlerTest.java

```diff
@@ -529,7 +530,7 @@ public void testCompletion_method_withLSPV2() throws JavaModelException{
 		assertTextEdit(5, 4, 6, "put", resolvedItem.getTextEdit());
 		assertNotNull(resolvedItem.getAdditionalTextEdits());
 		List<TextEdit> edits = resolvedItem.getAdditionalTextEdits();
-		assertEquals(3, edits.size());
+		assertEquals(2, edits.size());
```

Comment:
why do we suddenly have less textedits?

Reply:
Now, we convert an Eclipse MultiTextEdit to one VS Code WorkspaceEdit.